<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeLevelWidget</class>
 <widget class="QWidget" name="ChangeLevelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change item</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Actor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="PartyOperandWidget" name="operandParty_arg0" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Operation</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioInc">
        <property name="text">
         <string>Increase</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">groupOp_arg2</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="radioDec">
        <property name="text">
         <string>Decrease</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">groupOp_arg2</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Amount</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="ValueOperandWidget" name="operandAmount_arg3" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="check_arg5">
     <property name="text">
      <string>Show Level Up Message</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueOperandWidget</class>
   <extends>QWidget</extends>
   <header>ui/common/operand_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PartyOperandWidget</class>
   <extends>QWidget</extends>
   <header>ui/common/operand_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>helpRequested()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="groupOp_arg2"/>
 </buttongroups>
</ui>
